key/inputs/inputs_1_create.json

```diff
@@ -10,13 +8,11 @@
         "Sid": "Enable IAM User Permissions",
         "Effect": "Allow",
         "Principal": {
-          "AWS": "arn:aws:iam::${userAccount}:root"
+          "AWS": "arn:aws:iam::{{AWSAccountId}}:root"
```

Comment:
How does this work in other partitions ?

Reply:
It didn't, Good catch. I replaced 'aws' with '{{AWSPartition}}', which will grab the partition from an exported 'AWSPartition' CFN stack output.